.github/workflows/test-package.yml

```diff
@@ -134,7 +134,7 @@ jobs:
     - name: Install datacompy
       run: |
         python -m pip install --upgrade pip
-        python -m pip install .[tests,duckdb,polars,dask,ray]
+        python -m pip install .[tests,fugue]
```

Comment:
Should we have an environment for testing `datacompy` without `fugue` installed? Perhaps for an extra step of validation to make sure it's truly optional. Same for Snowflake and Spark I guess

Reply:
Oh yep, sorry didn't see that.